storage/src/lib.rs

```diff
@@ -261,6 +261,20 @@ pub enum CheckerError {
     #[error("The checker can only check persisted nodestores")]
     UnpersistedRoot,
 
+    #[cfg(feature = "ethhash")]
+    #[error(
+        "The node {key:?} at {address:#x} (parent: {parent:#x}) has a value but its path is not 32 or 64 bytes long"
```

Comment:
Is this key the debug representation of Path or the LowerHex representation? I think we want the latter.